tools-and-tests/k6/lib/grpc.js

```diff
@@ -0,0 +1,39 @@
+import {Stream} from "k6/net/grpc";
```

Comment:
Please add the license headers. I think i missed them on the files also
```suggestion
// SPDX-License-Identifier: Apache-2.0
import {Stream} from "k6/net/grpc";
```

Reply:
Agreed, what about [this comment](https://github.com/hiero-ledger/hiero-block-node/pull/1935#discussion_r2610245191)?